src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -308,6 +342,27 @@ internal override SpecialType SpecialType
             {
                 get { return GetSpecialType(_discriminator); }
             }
+
+            public override string ToString(string? format, IFormatProvider? provider)
```

Comment:
>override [](http://example.com/codeflow?start=19&length=8)

Can this `override` be `sealed`?

Reply:
Unfortunately not, because `ConstantValueDefault` overrides it so it can be formatted as `default(typename)`

Reply:
Could we remove the other overrides then? (It looks like there are a number of overrides that are a subset of this implementation.)